.github/workflows/playwright.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
         working-directory: ./support-e2e
 
       - name: Set outputs
+        if: ${{ failure() }}
```

Comment:
I'm not sure I understand this - my expectations is that the outputs would always be set? And then we just [read them on failure](https://github.com/guardian/support-frontend/pull/5820/files#diff-b710b539e104e4351f3e7598f94f434b8c7aaafdba88e57b3280582c718f7b4bR45).

I can't find any docs to suggest differently? But if it works, it's just something I am not getting.

Reply:
it seems to skip that step when the previous step failed
![image](https://github.com/guardian/support-frontend/assets/7304387/f3c2e572-3837-4d6a-9bd3-8cefea94c093)

Reply:
here are the docs, by default it's only run on success 
https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions

Reply:
Ah, so the `failure` is on the `steps` rather than the previous `step`. We live we learn - thanks for sharing!